devel/release.md

```diff
@@ -85,9 +85,22 @@ scripts/update-client.sh
 and commit changes (should be only version number changes) to the release branch.
 Name the commit something like "Update version constants for XXX release".
 
-```bash
-git push upstream $RELEASE_BRANCH
-```
+## Hot issues
```

Comment:
could you move it to the top? so people will see it clearly

Reply:
done